src/components/tables/Collections/index.tsx

```diff
@@ -47,25 +49,30 @@ function CollectionsTable() {
 
     return (
         <Box>
-            <EntityTable
-                noExistingDataContentIds={{
-                    header: 'collections.message1',
-                    message: 'collections.message2',
-                    docLink: 'collections.message2.docLink',
-                    docPath: 'collections.message2.docPath',
-                }}
-                columns={tableColumns}
-                query={liveSpecQuery}
-                renderTableRows={(data) => <Rows data={data} />}
-                setPagination={setPagination}
-                setSearchQuery={setSearchQuery}
-                sortDirection={sortDirection}
-                setSortDirection={setSortDirection}
-                columnToSort={columnToSort}
-                setColumnToSort={setColumnToSort}
-                header="collectionsTable.title"
-                filterLabel="entityTable.filterLabel"
-            />
+            <ZustandProvider
+                createStore={createSelectableTableStore}
+                storeName="Collections-Selectable-Table"
+            >
```

Comment:
Even tables that do not have selection available need to wrap the table in a store. I'd like for the entityTable to test if there is a store but it was a pain and this was faster.